mlflow/tracking/metic_value_conversion_utils.py

```diff
@@ -0,0 +1,163 @@
+import sys
+
+from mlflow.exceptions import MlflowException
+
+
+def is_module_imported(module_name: str) -> bool:
+    return module_name in sys.modules
+
+
+def __try_get_item(x):
```

Comment:
Nit: Can we use single underscore prefixes for these method names?

Reply:
Sure thing!